src/Lykke.Common.ApiLibrary/Swagger/FormFileUploadOperationFilter.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Linq;
+using Microsoft.AspNetCore.Http;
+using Swashbuckle.AspNetCore.Swagger;
+using Swashbuckle.AspNetCore.SwaggerGen;
+
+namespace Lykke.Common.ApiLibrary.Swagger
+{
+    /// <summary>
+    /// Replaces string input operation parameter which mapper by swagger
+    /// from IFormFile api parameter to file upload operation parameter.
+    /// 
+    /// IFormFile argument of controller's method should not be marked
+    /// by binding attribute such as [FromForm], [FromBody] or etc.
+    /// </summary>
+    public class FormFileUploadOperationFilter : IOperationFilter
+    {
+        public void Apply(Operation operation, OperationFilterContext context)
+        {
+            var descriptions = context.ApiDescription.ParameterDescriptions
+                .Where(x => x.ParameterDescriptor.ParameterType == typeof(IFormFile) && x.Source.Id == "FormFile")
+                .ToArray();
+
```

Comment:
Let's check, if IFormFile parameters are marked with one of binding attribute and throw exception if so. Exception message should be clear for application developer. Results of checking could be cached in ConcurrentDictionary